lib/elastic_apm/config.rb

```diff
@@ -49,7 +50,7 @@ class Config
     option :default_tags,                      type: :dict,   default: {}
     option :disable_send,                      type: :bool,   default: false
     option :disable_start_message,             type: :bool,   default: false
-    option :disabled_spies,                    type: :list,   default: %w[json]
+    option :disabled_instrumentations,         type: :list,   default: %w[json]
```

Comment:
In not including `:disabled_spies` in this list, aren't we dropping support for the `ELASTIC_APM_DISABLED_SPIES` environment variable as a result? I'm guessing we don't want to drop support until we actually remove the deprecated methods/option.

Reply:
Nice catch!